modules/core/src/main/scala/org/scalasteward/core/vcs/data/NewPullRequestData.scala

```diff
@@ -61,11 +64,29 @@ object NewPullRequestData {
         |${RepoConfigAlg.configToIgnoreFurtherUpdates(update)}
         |```
         |</details>
-        |
-        |${semVerLabel(update).fold("")("labels: " + _)}
+        |${appliedMigrations.getOrElse("")}
+        |labels: ${labels.mkString(", ")}
```

Comment:
@exoego I think we shouldn't add the string `labels:` if `labels` is empty.

Reply:
Updated in https://github.com/fthomas/scala-steward/pull/694/commits/d7f9771140614c5ebd9302c98668d6236ef5c53c